hack/builder/version.sh

```diff
@@ -1,3 +1,3 @@
-VERSION=$(date +"%y%m%d%H%M")-$(git rev-parse --short HEAD)
+VERSION=$(date +"%y%m%d")-$(git rev-parse --short HEAD)
```

Comment:
I would prefer to be very specific and not by accident move the tag. That will cause confusion.

Since we have no automation around this yet, feel free to change the push script itself. Just build in the push script itself and keep the version constant across build and push.

Something like this may work:


```diff
diff --git a/hack/builder/build.sh b/hack/builder/build.sh
index 2cfe90d1f..706d538a5 100755
--- a/hack/builder/build.sh
+++ b/hack/builder/build.sh
@@ -36,3 +36,4 @@ for ARCH in ${ARCHITECTURES}; do
     ${KUBEVIRT_CRI} pull --platform="linux/${ARCH}" quay.io/centos/centos:stream9
     ${KUBEVIRT_CRI} build --platform="linux/${ARCH}" -t "quay.io/kubevirt/builder:${VERSION}-${ARCH}" --build-arg SONOBUOY_ARCH=${sonobuoy_arch} --build-arg BAZEL_ARCH=${bazel_arch} -f "${SCRIPT_DIR}/Dockerfile" "${SCRIPT_DIR}"
 done
+echo ${VERSION}
diff --git a/hack/builder/publish.sh b/hack/builder/publish.sh
index c95ff23e4..cbe283ec3 100755
--- a/hack/builder/publish.sh
+++ b/hack/builder/publish.sh
@@ -17,6 +17,7 @@ cleanup() {
 }
 
 . ${SCRIPT_DIR}/version.sh
+VERSION=$($(dirname "$0")/build.sh)
 
 cleanup
```

Maybe a little bit better reworked, but just to illustrate the idea.